docs/docs/outputs/output-formats.md

```diff
@@ -81,3 +63,79 @@ When authoring a Go template the data source is Tracee's `trace.Event` struct, w
 Go template can utilize helper functions from [Sprig](http://masterminds.github.io/sprig/).
 
 For example templates, see [tracee/cmd/tracee-rules/templates](https://github.com/aquasecurity/tracee/tree/main/cmd/tracee-rules/templates).
+
+The following sections can be specified as part of go templates:
+
+```
+output:
+    # gotemplate:
+    #     template: /path/to/my_template1.tmpl
+    #     files:
+    #         - /path/to/output1.out
+    #         - /path/to/output2.out
+```
+
+### Webhook
+
+This sends events in json format to the webhook url
+
+Below is an example for configuring webhooks in the Tracee output section:
+
+```
+output:
+    # webhook:
+    #     - webhook1:
+    #         protocol: http
+    #         host: localhost
+    #         port: 8000
+    #         timeout: 5s
+    #         gotemplate: /path/to/template/test.tmpl
+    #         content-type: application/json
+    #     - webhook2:
+    #         protocol: http
+    #         host: localhost
+    #         port: 9000
+    #         timeout: 3s
+    #         gotemplate: /path/to/template/test.tmpl
+    #         content-type: application/json
+```
+
+Note: Please ensure that the respective fields will have to be uncommented.
+
+### Forward
+
+This sends events in json format using the Forward protocol to a Fluent receiver.
```

Comment:
I don't think it's actually json, and I don't think it really matters so better to remove the mention of json